lgc/test/ElfRelocationSize.lgc

```diff
@@ -0,0 +1,220 @@
+; RUN: lgc -mcpu=gfx1030 -extract=2 -o %t.fs.elf %s
+; RUN: lgc -mcpu=gfx1030 -extract=3 -other=%t.fs.elf -o %t.vs.elf %s
+; RUN: lgc -mcpu=gfx1030 -extract=1 -l %s -o %t.pipe.elf %t.vs.elf %t.fs.elf
+; RUN: llvm-readelf -W -S %t.pipe.elf | grep ".rel.text\|.rodata.cst32" | FileCheck %s
+
+; This test checks that no random extra bytes are generated after the relocations
+; in the relocation section (.rel.text). We check that we have the expected
+; number of bytes between this section and the one following it (.rodata.cst32).
+; Here we write 4 16-bit relocations, so we expect the following section to be at
+; OFFS+0x40 (or a bit more because of section alignment).
```

Comment:
Did you mean "byte" instead of "bit"?

Reply:
@s-perron sorry, I pushed again to use @kuhar suggested solution instead.